tests/test_config.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from src.config import *
+
+
+def test_is_config_exists():
+    assert is_config_exists("schema/config.json") == True
+    assert is_config_exists("schema/conf.json") == False
+
+
+def test_is_valid():
+    data = {
+        "ClusterMode": False,
+        "DeleteExistingTarget": True,
+        "Source": {
+            "Share": {
+                "AssumeRole": "arn:aws:iam::3211251512:role/db_restore_share_role",
+                "TargetAccount": "11223334544",
+                "SourceKmsKey": "alias/db_restore",
+                "TargetKmsKey": "alias/aws/rds",
+            },
+            "DBIdentifier": "dash-staging",
+        },
+        "Target": {
+            "AssumeRole": "arn:aws:iam::11223334544:role/db_restore_role",
+            "DBIdentifier": "dash-staging",
+            "VpcSecurityGroupIds": "${tf:db_global_security_group}",
+            "DBSubnetGroupName": "${tf:db_staging_private_subnet}",
+            "CopyTagsToSnapshot": True,
+            "DBInstanceClass": "db.t3.medium",
+            "PubliclyAccessible": False,
+            "Tags": [{"Key": "owner", "Value": "snapshot_restore"}],
+        },
+    }
+
+    status = is_valid(data)
+    assert status == True
```

Comment:
There is a nice concept of text fixture that can be kind of a reusable data shared by multiple tests, so for example
```suggestion
@pytest.fixture
def data():
    return {
        "ClusterMode": False,
        "DeleteExistingTarget": True,
        "Source": {
            "Share": {
                "AssumeRole": "arn:aws:iam::3211251512:role/db_restore_share_role",
                "TargetAccount": "11223334544",
                "SourceKmsKey": "alias/db_restore",
                "TargetKmsKey": "alias/aws/rds",
            },
            "DBIdentifier": "dash-staging",
        },
        "Target": {
            "AssumeRole": "arn:aws:iam::11223334544:role/db_restore_role",
            "DBIdentifier": "dash-staging",
            "VpcSecurityGroupIds": "${tf:db_global_security_group}",
            "DBSubnetGroupName": "${tf:db_staging_private_subnet}",
            "CopyTagsToSnapshot": True,
            "DBInstanceClass": "db.t3.medium",
            "PubliclyAccessible": False,
            "Tags": [{"Key": "owner", "Value": "snapshot_restore"}],
        },
    }

def test_is_valid():
    assert is_valid(data) == True
```
While the advantage is not obvious on this first test when you look at the next tests that take `data` without some parts, then it becomes obvious that the test could be bit more readable, as we specify what is missing and how that impacts `is_valid`, like for example `test_is_valid_missing_required_properties` could be:
```python
def test_is_valid_missing_required_properties():
    invalid_data = data.copy().pop("ClusterMode)
    assert is_valid(invalid_data) == False
```
Note how we get valid object from the fixture and then create a copy of it without the "ClusterMode" value.
You can use this pattern in every validation test that checks for ay of the missing keys